packages/web-react/src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useGridStyleProps } from './useGridStyleProps';
+import { SpiritGridProps } from '../../types';
+
+export const Grid = <T extends ElementType = 'div'>(props: SpiritGridProps<T>): JSX.Element => {
+  const { className, elementType: ElementType = 'div', children, ...restProps } = props;
```

Comment:
Apart from `Container`, custom element other than `div` for a `Grid` is a perfectly common use case 👍🏻. This typically includes lists: `ol`, `ul`, `dl`.